### PR TITLE
Some refactoring and make logos work

### PR DESCRIPTION
**Pull request by kelvin-chappell**
/cc @regiskuckaertz

### CODE REVIEW

File: commercial/app/views/jobs/jobsV2.scala.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
   omnitureId: Option[String],
   clickMacro: Option[String])(implicit request: RequestHeader)
 
+@import views.html.commercial.containerWrapper
+
```
Comment (regiskuckaertz) [inline commercial/app/views/jobs/jobsV2.scala.html]: Isn't `@import jobCard` missing here?

File: common/app/views/commercial/containerWrapper.scala.html [hunk 2]
```diff
@@ -28,6 +28,6 @@ <h1 class="adverts__title">@title</h1>
     </header>
     <div class="adverts__body">
         @innards
-        @for(badge        <- optBadge)   {<div class="adverts__badge js-badge">@badge</div>}
```
Comment (regiskuckaertz) [inline common/app/views/commercial/containerWrapper.scala.html]: That line remains necessary for the two cAPI components

**Comment by regiskuckaertz** [comment]
:+1: aside from minor comments, thanks a lot!

File: commercial/app/views/jobs/jobsV2.scala.html (hunk 1 above)
Comment (kelvin-chappell) [inline commercial/app/views/jobs/jobsV2.scala.html]: It was imported at the top of the file but I've moved it down now to make it more consistent.

File: common/app/views/commercial/containerWrapper.scala.html (hunk 2 above)
Comment (kelvin-chappell) [inline common/app/views/commercial/containerWrapper.scala.html]: Oops! Reinstated it now.

**Pull request merged**